dinghy-lib/src/ios/device.rs

```diff
@@ -667,6 +661,102 @@ fn launch_lldb_device<P: AsRef<Path>, P2: AsRef<Path>>(
     }
 }
 
+
+fn launch_app(
+    dev: &IosSimDevice,
+    app_args: &[&str]
+) -> Result<()> {
+    use std::io::{Read, Write};
+    let dir = ::tempdir::TempDir::new("mobiledevice-rs-lldb")?;
+    let tmppath = dir.path();
+    let mut install_path = String::from_utf8(
+        process::Command::new("xcrun")
+        .args(&["simctl", "get_app_container", &dev.id, "Dinghy"])
+        .output()?
+        .stdout,
+        )?;
+    install_path.pop();
+    let stdout = Path::new(&install_path).join("stdout").to_string_lossy().into_owned();
+    let stdout_param =
+        &format!("--stdout={}", stdout);
+    let mut xcrun_args : Vec<&str> = vec![
+        "simctl",
+        "launch",
+        "-w",
+        stdout_param,
+        &dev.id,
+        "Dinghy",
+    ];
+    xcrun_args.extend(app_args);
+    debug!("Launching app via xcrun using args: {:?}", xcrun_args);
+    let launch_output = process::Command::new("xcrun").args(&xcrun_args).output()?;
+    let launch_output = String::from_utf8_lossy(&launch_output.stdout);
+
+    // Output from the launch command should be "Dinghy: $PID" which is after the 8th character.
+    let dinghy_pid = launch_output.split_at(8).1;
+
+    // Attaching to the processes needs to be done in a script, not a commandline parameter or
+    // lldb will say "no simulators found".
+    let lldb_script_filename = tmppath.join("lldb-script");
+    let mut script = fs::File::create(&lldb_script_filename)?;
+    write!(script, "attach {}\n", dinghy_pid)?;
+    write!(script, "continue\n")?;
+    write!(script, "quit\n")?;
+    let output = process::Command::new("lldb")
+        .arg("")
+        .arg("-s")
+        .arg(lldb_script_filename)
+        .output()?;
+
+    let mut file = std::fs::File::open(stdout)?;
+    let mut test_contents = String::new();
+    file.read_to_string(&mut test_contents)?;
```

Comment:
Sorry, just thought about that: https://doc.rust-lang.org/std/fs/fn.read_to_string.html (no need to change it, just thought you may want to know it for next time).